rpm/droidmedia.spec

```diff
@@ -8,7 +8,7 @@
 
 Name:          droidmedia
 Summary:       Android media wrapper library
-Version:       0.0.0
+Version:       0.20170214.0
```

Comment:
This implies that from now on we should be modifying .spec nearly each time we tag?